doc/en/user/source/styling/sld-extensions/composite-blend.rst

```diff
@@ -0,0 +1,639 @@
+and 8 will be painted/composited on top of it, forming Merge3.
+Once Merge3 is ready, it will be painted/composited on top of the already fused Merge1/Merge2, generating
+the final map.
+
+An compositing and blending example
+-----------------------------------
+
+Let's say we want to draw topp:states so that the polygons are not filled with the population keyed
+colors, but only an inner border inside the polygon should appear, leaving the internal fully transparent.
+
+Using alpha-blending, this can be achived by creating a mask around the state borders with a thick
+stroke, and then using a "destination-in" alpha compositing.
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="ISO-8859-1"?>
+    <StyledLayerDescriptor version="1.0.0" xmlns="http://www.opengis.net/sld" xmlns:ogc="http://www.opengis.net/ogc"
+                           xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:gml="http://www.opengis.net/gml"
+                           xsi:schemaLocation="http://www.opengis.net/sld">
+      <NamedLayer>
+        <Name>USA states population</Name>
+        <UserStyle>
+          <Name>population</Name>
+          <Title>Population in the United States</Title>
+          <Abstract>A sample filter that filters the United States into three
+            categories of population, drawn in different colors</Abstract>
+          <FeatureTypeStyle>
+            <!-- Usual states rules, skipped for brevity -->
+          </FeatureTypeStyle>
+          <FeatureTypeStyle>
+            <Rule>
+              <LineSymbolizer>
+                <Stroke>
+                  <CssParameter name="stroke-width">10</CssParameter>
+                  <CssParameter name="stroke">#000000</CssParameter>
+                </Stroke>
+              </LineSymbolizer>
+            </Rule>
+            <VendorOption name="composite">destination-in</VendorOption>
+          </FeatureTypeStyle>
+          <FeatureTypeStyle>
+            <!-- The black border and labels rules, skipped for brevity -->
+          </FeatureTypeStyle>
+        </UserStyle>
+      </NamedLayer>
+    </StyledLayerDescriptor>
+
+Visually, this is the destination:
+
+.. figure:: composite-images/states.png
+
+This is the source (the mask):
+
+.. figure:: composite-images/states-border.png
+
+And this is the result of the composition:
+
+.. figure:: composite-images/states-border-composite.png
+
+Now, what happens if someone makes a WMS call in which the Natural Earth background is set as
+the first layer, and states with the above style, as the second? The following:
+
+.. figure:: composite-images/ne-states-border-composite1.png
+
+The backdrop with the DEM and bathymetry is hardly visible, because it has been cut by the mask.
+This is not what we want to achieve, and shows the risks of using alpha compositing without care
+in a WMS setting.
```

Comment:
Great docs starting off with a negative example. It does demonstrate the risks clearly....